Relations/Views/Home/AddRelation.cshtml

```diff
@@ -1,4 +1,4 @@
-﻿@model Relation
+﻿@model CreateRelationViewModel
     <form asp-controller="Home" asp-action="AddRelation" method="post">
```

Comment:
asp-controller="Home" ? 
it should be asp-controller="Relation"
and change path to it: Relations/Views/**Relation**/AddRelation.cshtml